### PR TITLE
feat: add fal deploy

### PR DESCRIPTION
**Pull request by isidentical**

### CODE REVIEW

File: projects/fal/src/fal/app.py [hunk 1]
```diff
@@ -64,15 +65,33 @@ def initialize_and_serve():
...
+
 @mainify
 class App(fal.api.BaseServable):
     requirements: ClassVar[list[str]] = []
     machine_type: ClassVar[str] = "S"
-    host_kwargs: ClassVar[dict[str, Any]] = {}
+    host_kwargs: ClassVar[dict[str, Any]] = {
+        "_scheduler": "nomad",
+        "_scheduler_options": {
+            "storage_region": "us-east",
```
Comment (turbo1912) [inline projects/fal/src/fal/app.py:83]: does this mean fast lambda cluster always?

Comment (isidentical) [reply projects/fal/src/fal/app.py:83]: no. it means anything in east which has other stuff too (like a6000 machines)

File: projects/fal/src/fal/cli.py [hunk 2]
```diff
@@ -278,7 +298,54 @@ def load_function_from(
...
+    user_id = _get_user_id()
+    isolated_function, app_name = load_function_from(serverless_host, str(file_path))
+    id = serverless_host.register(
+        func=isolated_function.func,
+        options=isolated_function.options,
+        application_name=app_name,
+        metadata=isolated_function.options.host.get("metadata", {}),
+    )
+
+    if id:
```
Comment (mederka) [inline projects/fal/src/fal/cli.py:339]: Should we let the the user know if `id` is not there?

**Review by mederka — APPROVED** [review]
Do we have corresponding docs in UI?

**Comment by efiop** [comment]
For the record: I'm stealing^W borrowing this to incorporate into https://github.com/fal-ai/fal/pull/206

image

**Comment by efiop** [comment]
Closing in favor of https://github.com/fal-ai/fal/pull/206

**Pull request closed without merging**